korman/ui/modifiers/gui.py

```diff
@@ -88,6 +88,9 @@ def row_alert(prop_name, **kwargs):
 
         layout.separator()
         layout.prop(modifier, "link_type")
+        if modifier.link_type == "kOriginalBook":
+            layout.prop(modifier, "shareable")
+            layout.prop(modifier, "shr_region")
```

Comment:
```suggestion
            layout.prop(modifier, "share_region")
```

Reply:
Something I _always_ tend to forget. 😜 Fixed.